### PR TITLE
Upgrade to bugsnag-js 7.x (latest)

### PR DESCRIPTION
**Pull request by pdud**
This PR gets us running on the latest bugsnag-js! 🎉

💥Breaking change💥

#### Upgrading

1. Rename `notifyReleaseStages`/`BUGSNAG_NOTIFY_RELEASE` to `enabledReleaseStages`/`BUGSNAG_ENABLED_RELEASE` - (in sync with bugsnag-js)

2. `endpoint`/ `BUGSNAG_ENDPOINT` => `endpoints`/`BUGSNAG_ENDPOINTS`  - both are now required in bugsnag-js 7.x

3. Rename `getMetaData` =>  `getMetadata` in `app/utills/bugsnag.js` (in sync with bugsnag-js)

**note:** Bugsnag will use the current client when you use the default import  `import Bugsnag from 'bugsnag'` - This is how bugsnag-js now works.

So usage is simplified due to no longer having to worry about the current client.

Let me know if I can make any changes! Thanks!

### CODE REVIEW

**Comment by pdud** [comment]
hey @jderr-mx I know you must be super busy! I was wondering what you thought of this PR?

**Comment by jderr-mx** [comment]
> hey @jderr-mx I know you must be super busy! I was wondering what you thought of this PR?

Hi @pdud My apologies for the delay. Looks like I need to make sure I get notifications when PRs are opened. 
I'll take a look and assuming all is well I'll release it as 3.0.0 since there are breaking changes.

File: README.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ There are two ways to configure `ember-cli-bugsnag`:
 {
```
Comment (jderr-mx) [inline README.md:19]: I plan to bump the major version to 3.0, can you add a section to the readme that mentions 3.0 contains breaking changes?

**Comment by pdud** [comment]
Thanks for reviewing @jderr-mx! I've added a section to the readme as requested. Let me know if there is any changes you'd like to see.

**Comment by jderr-mx** [comment]
Awesome thank you!

**Pull request merged**